.gitignore

```diff
@@ -0,0 +1,2 @@
+.vscode
+.idea
```

Comment:
This sort of thing is coming from your editor - it should go in a global git-ignore file on your machine, rather than in the repository.  [This looks like a helpful link](https://gist.github.com/subfuzion/db7f57fff2fb6998a16c).